lib/setting_types.rb

```diff
@@ -0,0 +1,9 @@
+module SettingTypes
+    def board_size
```

Comment:
Methods like this that just return a static value can just be declared as constants

```ruby
module SettingTypes
  BOARD_SIZE = :board_size
  OPPONENT_TYPE = :opponent_type
end
```